### ISSUE TITLE
GrabButton interaction broken 

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#151]
Caused by commits from this morning due to https://github.com/phetsims/scenery/issues/898. I think it may be as simple as converting listeners to use the new `inputListener` approach, but I'm unsure. Tagging @jessegreenberg so he is aware.

**Comment by zepumph** [comment]
For reference. The issue is that you tab to the book. Then grab it, and when you tab away it goes back to the grab button, instead of on to the next focusable item.

For more reference, here are the shas to check out to take you back to when it was working correctly:
joist: 88658dbb43ae78d16ce698545f604d6173176eeb
scenery: 6d899bca621aa348d471d15da410ca647deb41f9

**Comment by zepumph** [comment]
This is actually sort of a pain. The solution is to have `focus` and `blur` events be added with `addInputListener` instead of the a11y alternative. Normally that would be easy, but in this type we are keeping track of all listeners and swapping them out between the grab and drag versions of the node's a11y interaction. In order to get this working again from master, here is what I would need to do either support keydown, click, and keyup in a sustainable way through Input.js or:

1. split the focus/blur listeners out to separate objects.
2. Somehow keep track of when a listener should be added via `addInputListener` or `addAccessibleInputListener` in the list of listeners `listenersForDrag` and `listenersForGrab`.
3. Change usages of the sites that were using the above array as options
4. extend that branching logic out to the adding and removal of the input listeners. Currently there is a method like this:
```js
    addInputListeners( listeners ) {
      for ( let i = 0; i < listeners.length; i++ ) {
        const listener = listeners[ i ];
        if ( !this.node.hasAccessibleInputListener( listener ) ) {
          this.node.addAccessibleInputListener( listener );
        }
      }
    }

```

@jessegreenberg how hard would it be to do a sprint and support all events needed through scenery? I know we have been going slowly for a reason, but I also see the benefit (since we are at this crossroads), in just plowing ahead. It would be better to spend the time on the future instead of branch/workaround drama. Adding to shorts meeting tomorrow.

**Comment by zepumph** [comment]
This has been taken care of by commits in phetsims/scenery#898. We decided to go forward with converting all events over to the system in scenery/Input.js.
